xocobo12/niuro-test
# DB Connection, gitignore and test_db with mock data

### Short description
- `db.py` file added and stablish connection using SQLite
- Added file `test_db.py` to test the connection with DB
- Modified requirements.txt work with streamlit and sqlite
- Created tables users and added mock data to test
- Added .`env.example` and starting using environment variables instead of hardcoded values

### How to test it 

Execute: `streamlit run test_db.py
`
You should see something like this

![image](https://github.com/user-attachments/assets/2e0a2ecf-1105-4fb4-ae6f-fc20d976327e)
